src/transliteration/ui/line-tokens.tsx

```diff
@@ -65,6 +65,7 @@ export class LineToken {
   token: LemmatizableToken
   lemma: DictionaryWord[] | null = null
   siglum: string | null = null
+  type = 'LineToken'
```

Comment:
`type: string = 'LineToken'`

Reply:
Normally yet, but in this case the linter complains `Type string trivially inferred from a string literal, remove type annotation`.